codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/EntityServiceSourceGenerator.scala

```diff
@@ -148,7 +148,7 @@ object EntityServiceSourceGenerator {
       case eventSourcedEntity: EventSourcedEntity =>
         eventSourcedEntitySource(service, eventSourcedEntity, packageName, className, interfaceClassName, entityType)
       case valueEntity: ValueEntity =>
-        valueEntitySource(service, valueEntity, packageName, className, interfaceClassName, entityType)
+        ValueEntitySourceGenerator.valueEntitySource(service, valueEntity, packageName, className, interfaceClassName)
```

Comment:
Starting to move thing out to its own generator. For the moment just calling the method in new `ValueEntitySourceGenerator`